### DISCUSSION TITLE
Support for Ubuntu 16.04

### DISCUSSION THREAD

**Discussion by mqtik** [utmapp/UTM#3430] [General]
Hi there, first of all, thanks for such an amazing project, runs really well in my mac.

I was wondering if you guys have an image for Ubuntu 16.04. I've seen 20.04, and 14.04, but no 16.04

Thanks!

**Comment by osy** [comment]
The prebuilt images are examples. You can easily install your own Ubuntu with the right install ISO.

**Comment by AwlsomeAlex** [comment]
Platform has a little bit of significance. If you're using a device with Apple Silicon, Ubuntu 16.04 ARM won't boot. It requires 64K page size, whereas the M1 only supports 16K page size. You could run the x86 version on Apple Silicon but with a performance hit. Otherwise if you're running an Intel device, you can download the install ISO [here](https://releases.ubuntu.com/16.04/).

**Reply by erzakiev** [reply]
Yes 16.04 ARM64 doesn't boot on M1 silicon :(
